spec/controllers/orders_controller_spec.rb

```diff
@@ -143,7 +166,11 @@
       session[:order_id] = test_order.id
     end
 
-    let(:checkout_buyer_params) { { order: { buyer_name: "My name", buyer_email: "my_email@example.com", buyer_address: "123 Example St, Cityville, State 12345", buyer_card_short: "1234", buyer_card_expiration: future_date } } }
+    let(:checkout_buyer_params) { { order: {
```

Comment:
Sure. If you're wondering why I did that:
* We have a line length limit in the classroom style guide. 120 characters.